submissions/Eugene-CG/memory-pair-game/index.js

```diff
@@ -0,0 +1,92 @@
+const cardsList = document.querySelector(".cards__list");
+let clicksStorage = [];
+
+const startGame = () => {
+  let cards = [
```

Comment:
`const`. Variable is not changed anywhere.

Always use `const`, except situations in which you will rewrite value in it.

Reply:
I cant declare clicksStorage variable with const keyword bcs I change clicksStorage later(69 line)